components/Slider/src/MDCSlider.h

```diff
@@ -47,22 +49,35 @@ IB_DESIGNABLE
 
  Default color is blue.
  */
-@property(nonatomic, strong, null_resettable) UIColor *color;
+@property(nonatomic, strong, null_resettable) UIColor *color UI_APPEARANCE_SELECTOR;
```

Comment:
Not sure why we chose these properties to be `null_resettable` historically, but In general, all of Apple's UI_APPEARANCE_SELECTOR properties are `nullable` so maybe worth looking into this for future revisions.